src/core/utils/addons.js

```diff
@@ -43,16 +44,19 @@ export const getErrorMessage = ({
 export const getFileHash = ({
   addon,
   installURL,
+  version,
 }: {|
   addon: AddonType,
   installURL: string,
+  version?: AddonVersionType,
```

Comment:
I thought you wanted to use `| void` instead of `?:`?

Reply:
FTR it was @kumar303, not me that had an issue with it, but yes, this looks like a case where `AddonVersionType | void` would be better. Although looking closer, it seems like `version` could be required as I don't see any cases where it wouldn't exist when calling this code. I'll look into that too.